training/distributed_training/mpi_on_sagemaker/intro/mpi_demo.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>When you introduce <code>distribution</code>  it would be a good opportunity to mention the other distribution types.</p><ol><li>data parallel: https://sagemaker.readthedocs.io/en/stable/api/training/smd_data_parallel.html</li><li>model parallel: https://sagemaker.readthedocs.io/en/stable/api/training/smd_model_parallel_general.html#required-sagemaker-python-sdk-parameters</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>It might look like a distraction to introduce too many distribution strategies before this cell. will do it in the conclusion section, or I can add another section at the end for "Looking further" </p>